src/main/scala/rps/Game.scala

```diff
@@ -1,15 +1,16 @@
-package rps;
+package rps
 
 object Game {
   def play(): Unit = {
 
     /* Ask for user input */
-    val userMove = readLine(
-      "What's your move?\n" +
-        "0: " + returnMoveIcon("0") + "\n" +
-        "1: " + returnMoveIcon("1") + "\n" +
-        "2: " + returnMoveIcon("2") + "\n\n" +
-        "> ")
+    val userMove = readLine(s"""
+    #What's your move?"
+    #0: ${returnMoveIcon("0")}
+    #1: ${returnMoveIcon("1")}
+    #2: ${returnMoveIcon("2")}
+    #
+    #> """.stripMargin('#'))
```

Comment:
this is 👍 but there's no need to define a custom separator. The default one is `|` and it works like

```scala
val userMove = readLine(s"""
    |What's your move?"
    |0: ${returnMoveIcon("0")}
    |1: ${returnMoveIcon("1")}
    |2: ${returnMoveIcon("2")}
    |
    |> """.stripMargin)
```

which is a bit more readable :)

Reply:
Gotcha! I wanted to try a new one, but in the next step I'll change it!